openquake/hazardlib/tests/geo/surface/gridded_test.py

```diff
@@ -31,7 +31,7 @@
 
 
 class GriddedSurfaceTestCase(unittest.TestCase):
-
+    
     def setUp(self):
```

Comment:
Now I think I understand the value of the angle that you were getting before which I believe was more correct than the value you have now in the dip angle test (how did you get that?). The point you use to define the gridded surface are on an almost horizontal plane. This is ok-ish for a test. Best would be to take for example a simple of complex fault, create a cloud of points around it and do the interpolation on this dataset.